guides/image-sources/provisioning-image-userdata.plantuml

```diff
@@ -0,0 +1,48 @@
+@startuml
+
+!include foreman.pstyle
+
+title Image-based provisioning with cloud-init configuration
+
+actor       User
+participant "Provisioned\ninstance" as Host
+participant "Foreman" as Foreman
+participant "Foreman\nProxy" as Proxy
+participant "Infrastructure\ncloud" as Cloud
+participant DNS
+!if ($puppet)
+participant "Puppet\nserver" as Puppet
+!endif
+
+note over Foreman : Has an image\nwith the //User Data// flag
+note over Host : Powered off
+
+== Create host in Foreman ==
+
+User -> Foreman : Create host\nwith a cloud resource and the image
+Foreman -> Cloud : Create a new instance\nwith user-data info
```

Comment:
@stejskalleos Not sure if this line is accurate and what "user-data info" means.

Reply:
Good enough